phpunit.xml.dist

```diff
@@ -20,4 +20,4 @@
         <const name="GOOGLESITEKIT_TESTS" value="1" />
         <const name="WP_HTTP_BLOCK_EXTERNAL" value="1" />
     </php>
-</phpunit>
\ No newline at end of file
+</phpunit>
```

Comment:
This change isn't needed, is it? Let's revert it then.

Reply:
Reverted in https://github.com/google/site-kit-wp/pull/3495/commits/f3e2f5505126f014e43b6678e1781a6dcf8e4338